auth-web/src/views/AcceptInvite.vue

```diff
@@ -49,7 +49,12 @@ export default class AcceptInvite extends Mixins(NextPageMixin) {
   private async mounted () {
     await this.getUserProfile('@me')
     await this.syncOrganizations()
-    this.accept()
+    // Check to make sure this user is not already a member of a team
+    if (this.organizations.length > 0) {
+      this.$router.push('/duplicateteam')
```

Comment:
If possible ,add this url to pages enum..so that one single place for all urls

Reply:
Done.  Thanks!